joomla/joomla-cms
# [4.2] Remove ID attribute from mod_custom

### Summary of Changes

It redo of #32980, but without use of ID attribute


### Testing Instructions
Select background Image for mod_custom,
And check module on the site, also the module HTML markup


### Actual result BEFORE applying this Pull Request
Background visible, module have an ID attribute


### Expected result AFTER applying this Pull Request
Background visible, module do not have an ID attribute


### Documentation Changes Required
none
